doc/self-hosting.md

```diff
@@ -176,3 +176,20 @@ Set the `REDIRECT_URI` environment variable:
 ```sh
 REDIRECT_URI=http://my-custom-shields.s3.amazonaws.com/
 ```
+
+Sentry
+------
+
+In order to enable integration with [Sentry](https://sentry.io), you need your own [Sentry DSN](https://docs.sentry.io/quickstart/#configure-the-dsn). It’s an URL in format `https://{PUBLIC_KEY}:{SECRET_KEY}@sentry.io/{PROJECT_ID}`.
+
+### How to obtain the Sentry DSN
+
+1. [Sign up](https://sentry.io/pricing/) for Sentry
+1. Log in to Sentry
+1. Create a new project for Node.js
+1. You should see [Sentry DSN](https://docs.sentry.io/quickstart/#configure-the-dsn) for your project. Sentry DSN can be found by navigating to \[Project Name] -> Project Settings -> Client Keys (DSN) as well.
+
+Start the server using the Sentry DSN:
+```
+SENTRY_DSN=https://xxx:yyy@sentry.io/zzz sudo node server
```

Comment:
Could you document also relying on private/secret.json?

Reply:
done